### DISCUSSION TITLE
样式baseon怎么拿到在dll文件里的样式

### DISCUSSION THREAD

**Discussion by AIKA1024** [BYJRK/DotNet-Discussions#49] [WPF 开发]
我现在导入了一个叫wpfui的ui库，我想给listbox里的listboxitem修改一下contextmanu，我用了style修改的话我listboxitem的样式就全没了，用baseon的话不知道怎么拿到dll里的样式。
或者有更好的办法修改contextmanu（除了在c#代码里一个个修改，这个我不太能接受）
如果有的话 请大佬指点一下

**Comment by BYJRK** [comment]
如果 dll 提供了默认样式的名字（通常名字形如 `DefaultListBoxItemStyle` 之类），那么就 basedon 这个即可

否则可以这样写：`Style="{StaticResource {x:Type ListBoxItem}"`
